test/test_meta.py

```diff
@@ -690,12 +691,6 @@ def run_meta_crossref(
 meta_function_device_expected_failures_only_outplace = defaultdict(dict)
 meta_function_device_skips = defaultdict(dict)
 
-meta_function_device_expected_failures['cpu'] = {
-    torch.native_batch_norm: {bf16},
-    torch._native_batch_norm_legit: {bf16},
-    torch.native_layer_norm: {bf16},
-}
-
```

Comment:
Why were these removed?

Reply:
I can put the fixes in a different PR, I made the fixes for these in decompositions.py

Reply:
Yes, please separate out the fixes from the new tests. It makes it easier to tell what changed and why